WeakAurasOptions/AuthorOptions.lua

```diff
@@ -1147,13 +1147,26 @@ end
 
 local function allChoicesAreDefault(data)
   for _, option in ipairs(data.authorOptions) do
-    if option.key ~= nil and option.default ~= nil and option.default ~= data.config[option.key] then
+    if option.key ~= nil and option.default ~= nil and (option.default ~= data.config[option.key] or compareTables(option.default, data.config[option.key])) then
```

Comment:
This doesn't work because the logic is the wrong way around. You want to check if the tables are different.

Also this will not work if you are comparing two indentical values, as in that case compareValues is called with two values, and calling pairs on that will result in a error.